Makefile

```diff
@@ -18,13 +18,8 @@ shellcheck: build
 vagrant:
 	vagrant up
 
-docker-machine: vagrant
-	docker-machine create -d generic \
-	  --generic-ssh-user $(shell vagrant ssh-config | grep ' User ' | cut -d ' ' -f 4) \
-	  --generic-ssh-key $(shell vagrant ssh-config | grep IdentityFile | cut -d ' ' -f 4) \
-	  --generic-ip-address $(shell vagrant ssh-config | grep HostName | cut -d ' ' -f 4) \
-	  --generic-ssh-port $(shell vagrant ssh-config | grep Port | cut -d ' ' -f 4) \
-	  image-builder-rpi
+docker-setup: vagrant
+	export DOCKER_HOST=tcp://127.0.0.1:2376
```

Comment:
Can we use port 2375 for the unprotected TCP port. 2376 is misleading.

Reply:
I've used 2376 not to conflict with default docker for mac 2375, but happy to change it

Reply:
Docker for Mac uses the default Unix socket, so 2375 is free

Reply:
ack! will change